moja-kuhinjica-frontend/src/components/button/HeaderButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styles from './HeaderButton.module.scss'
+
+interface IHeaderButtonProps {
+    active: number
+    setActive: (param: number) => void
+    buttonNumber: number
+    content: string
+}
+export const HeaderButton = ({
+    active,
+    setActive,
+    buttonNumber,
+    content,
+}: IHeaderButtonProps) => {
+    return (
+        <button
+            className={
+                active === buttonNumber
```

Comment:
This comparison can be done in parent component, so you don't really need this comparison here

Reply:
Done

Reply:
This is not done